### PR TITLE
Create numpy cheatsheet

### PR DESCRIPTION
**Pull request by ritwikdalmia**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/Numpy-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+- Arithmetic Operations
+|Operation type | Syntax|
+|-------------|----------|
+| Addition| np.add(a,b)|
+| Subtraction| np.subtract(a,b)|
+| Multiplication| np.multiply(a,b)|
+| Division| np.divide(a,b)|
+| Exponentiation| np.exp(a)|
+| Square Root| np.sqrt(b)|
```
Comment (crescentpartha) [inline CheatSheets/Numpy-cheatsheet.md:166]: The table is broken.

**Comment by ritwikdalmia** [comment]
please merge this PR

File: README.md [hunk 2]
```diff
@@ -50,10 +50,14 @@ A Collection of programming [CheatSheets](./CheatSheets/) for developers to quic
...
 19. [php-cheatsheet.md](./CheatSheets/php-cheatsheet.md)
+
+
 20. [bash-cheatsheet.md](./CheatSheets/bash_cheatsheet.md)
 21. [kotlin-cheatsheet.md](./CheatSheets/kotlin-cheatsheet.md)
 22. [vim-editor-cheatsheet.md](./CheatSheets/vim-editor-cheatsheet.md)
-
+23. 20. [pandas-cheetsheet.md](./CheatSheets/pandas-cheetsheet.md)
+24. [numpy-cheatsheet.md]((./CheatSheets/numpy-cheetsheet.md))
+25.
```
Comment (crescentpartha) [inline README.md:60]: You broke the entire list.

File: CheatSheets/Numpy-cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+|-------------|----------|
+|`a[0]=5` | Assigns array element on index 0 the value 5|
+| `a[2,3]=1` | Assigns array element on index [2][3] the value 1|
+
+- Subseting
+
+|keywords | description|
+|-------------|----------|
+| `a[2]`| Returns the element of index 2 in array a.|
+|`a[3,5]` | Returns the 2D array element on index [3][5]|
```
Comment (crescentpartha) [inline CheatSheets/Numpy-cheatsheet.md:143]:
```suggestion
- Subseting

|keywords | description| Action |
|-------------|----------| --------- |
| `a[2]`| Returns the element of index 2 in array a.| `Subseting` |
|`a[3,5]` | Returns the 2D array element on index [3][5]| `Subseting` |
```
Don't need any listing between the two tables. Directly include in the table. 
|keywords | description| Action |
|-------------|----------| --------- |
| `a[2]`| Returns the element of index 2 in array a.| `Subseting` |
|`a[3,5]` | Returns the 2D array element on index [3][5]| `Subseting` |

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Modification needed.

**Comment by ritwikdalmia** [comment]
conflict solved

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**